smart_open/daos.py

```diff
@@ -0,0 +1,311 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2022 Hewlett Packard Enterprise Development LP.
```

Comment:
@piskvorky We need the copyright transferred to you explicitly, right?

Reply:
Yes. Managing copyright from a multitude of authors would be a nightmare. Too much legal trouble for zero (open source) gain.

Reply:
@sridharbalachandriah Please see the other files for examples of what the copyright header should be.

Reply:
@mpenkov ,@piskvorky, 

Would the copyright attached below suffice? 

# -*- coding: utf-8 -*-
#
# Copyright (C) 2022 Radim Rehurek <radim@rare-technologies.com>
# Copyright 2022 Hewlett Packard Enterprise Development LP.
#
# This code is distributed under the terms and conditions
# from the MIT License (MIT).
#

Reply:
No, sorry. We want full control over `smart_open`, and having to coordinate with multiple copyright owners complicates that. We're happy to acknowledge your authorship of course, so a header like `Original auhor(s): X Y Z` is perfectly fine.

Reply:
@piskvorky , Thank you for the inputs.

Would the content outlined below work?

**
Copyright (C) 2022 Radim Rehurek [radim@rare-technologies.com](mailto:radim@rare-technologies.com)
Original author(s): X Y Z, Hewlett Packard Enterprise Development LP.

This code is distributed under the terms and conditions
from the MIT License (MIT).**

Reply:
Yes, thanks.